DiamondLightSource/hyperion
# Panda grid scans: Zebra handles fast shutter control

Fixes #1144 
This PR also adds a counter to the panda, which keeps track of how many eiger triggers have been sent in a scan.

### To test:
1. Check changes are sensible and tests pass
2. Confirm any relevant documentation has been updated (Is it just wiring diagrams which need to change?)
